primer.md

```diff
@@ -112,11 +112,36 @@ specifications of how to serialize the event in different formats and
 transports (e.g. JSON and HTTP).
 
 ### Non-Goals
-The following will not be part of the specification:
+The following are considered beyond the scope of the specification:
 * Function build and invocation process
 * Language-specific runtime APIs
 * Selecting a single identity/access control system
 
+The following are features that the working group has explicitly decided
+should not be part of the specification:
+* **routing information** The CloudEvents spec will never include routing
+   information (e.g. a Pub/Sub topic over which the message is delivered).
```

Comment:
I think the idea is to avoid attributes that are specific to transports and I support that intention. Nevertheless routing happens on many layers.

Reply:
I'm leaning towards resolving this before we merge, but @inlined what do you think?

Reply:
My quote from above was from the design goals section of the primer, which is just above the non-goals section. So in the goals we state that the event context contains information that is needed to route the event, and in the non-goals we say that the CloudEvents spec will never include routing information. This might confuse someone reading through this for the first time.